jstp/src/main/java/com/metarhia/jstp/connection/SessionRestorationPolicy.java

```diff
@@ -33,10 +36,20 @@ public void onTransportAvailable(Connection connection, String appName, String s
     }
   }
 
+  /**
+   * Checks if reconnection needed when transport is ready
+   *
+   * @return is reconnection needed when transport is ready
```

Comment:
Please change the style of a return statement to be consistent with the ones above.